pkg/codegen/customresourcegenerator.go

```diff
@@ -16,18 +16,26 @@ package codegen
 
```

Comment:
Most of the schemagen changes occur here. This is where we flatten OpenAPI specs.